inc/classes/admin/settings/class-page.php

```diff
@@ -870,6 +870,14 @@ private function media_section() {
 			]
 		);
 
+		$rocket_maybe_disable_lazyload_plugins = '';
+		if ( rocket_maybe_disable_lazyload() ) {
+			$rocket_maybe_disable_lazyload_plugins = __( 'Autoptimize', 'rocket' ) . ', ';
+		}
+
+		$rocket_maybe_disable_lazyload_plugins = apply_filters( 'rocket_maybe_disable_lazyload_helper', $rocket_maybe_disable_lazyload_plugins );
+		$rocket_maybe_disable_lazyload_plugins = rtrim( $rocket_maybe_disable_lazyload_plugins, ', ' );
```

Comment:
Instead of adding `', '` at the end of `$rocket_maybe_disable_lazyload_plugins` and then trimming, I have a proposal:
Make `$rocket_maybe_disable_lazyload_plugins` an array and push plugin names into it. Then use this magic function: [wp_sprintf_l()](https://wpseek.com/function/wp_sprintf_l/).
You get the final string like this:
```php
$rocket_maybe_disable_lazyload_plugins = wp_sprintf_l( '%l', $rocket_maybe_disable_lazyload_plugins );
```

Depending on the number of items, it will output:
- 1: `plugin 1`,
- 2: `plugin 1 and plugin 2`,
- 3+: `plugin 1, plugin 2, and plugin 3`.

It’s a nice way to localize lists without headache, moreover arrays are easier to manipulate :)
What do you think?

Reply:
Nice magic. It's much more clear now.